app/models/user.rb

```diff
@@ -82,7 +82,11 @@ def client_admin?
   end
 
   def admin?
-    self.has_role?('admin')
+    has_role?('admin')
+  end
+
+  def not_admin?
+    !admin?
   end
```

Comment:
So I'm curious about `not_admin` vs `!admin` -- one is less typing (which seems to be the Ruby way?) and the other more legible. Do you prefer readability over concision?